lib/streamlit/stats.py

```diff
@@ -92,13 +91,14 @@ def options(self):
         self.finish()
 
     def get(self) -> None:
-        metric_type = "# TYPE st_cache_memory_bytes gauge"
-        metric_unit = "# UNIT st_cache_memory_bytes bytes"
+        metric_type = "# TYPE cache_memory_bytes gauge"
+        metric_unit = "# UNIT cache_memory_bytes bytes"
```

Comment:
@AnOctopus this looked like a typo to me (`st_cache_memory_bytes` instead of `cache_memory_bytes`) but please confirm